### PR TITLE
add Circuitbreaker webUI

### LINKED ISSUE #3598: [feat] expose the circuitbreaker webui
https://twitter.com/joostjgr/status/1612511153950711830

https://github.com/lightningequipment/circuitbreaker

### PR DESCRIPTION
**Pull request by openoms**
fix #3598 

- [x] add the webui link to the menu entry

As it is not an authenticated endpoint won't create a Tor hidden service for it.

### CODE REVIEW

**Review by rootzoll — APPROVED** [review]
LGTM - will run test before merge

File: home.admin/assets/nginx/sites-available/circuitbreaker_ssl.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## circuitbreaker_ssl.conf
+
+server {
+    listen 9235 ssl http2;
```
Comment (rootzoll) [inline home.admin/assets/nginx/sites-available/circuitbreaker_ssl.conf]: think this one needs to be 9236 - not 9235

Comment (openoms) [reply home.admin/assets/nginx/sites-available/circuitbreaker_ssl.conf]: fixed in [bb679fe](https://github.com/rootzoll/raspiblitz/pull/3599/commits/bb679fe0eda92712376f7703d350ee5455d511e0)

**Comment by rootzoll** [comment]
OK test install worked now :)

**Pull request merged**